system-tests/project-fixtures/react/src/mount.cy.jsx

```diff
@@ -24,4 +24,11 @@ describe('mount', () => {
       cy.get('[data-cy-root]').children().should('have.length', 1)
     })
   });
+
+  it('does not error when rendering primitives', () => {
+    cy.mount('Hello World')
+    cy.mount(5)
+    cy.mount(null)
+    cy.mount(undefined)
```

Comment:
Tests look good, should we write these in npm/react where possible?

Reply:
Would like to but this way allows testing across react17 and react18 which `npm/react` doesn't allow for